test/jdk/java/awt/print/PrinterJob/PrintLatinCJKTest.java

```diff
@@ -21,57 +21,72 @@
  * questions.
  */
 
-/**
+/*
  * @test
- * @bug 800535 8022536
+ * @bug 8022536
```

Comment:
Why is [800535](https://bugs.openjdk.java.net/browse/JDK-8008535) removed? It's the bug this test is written for.

The additional bug [8022536](https://bugs.openjdk.java.net/browse/JDK-8022536) may cause NPE on Linux if a (default) remote printer is unavailable.